app/views/landing_page/blocks/_heading.html.erb

```diff
@@ -1,12 +1,14 @@
 <%
   heading_level = block.data["heading_level"] || 2
+  font_size = block.data["font_size"] || 'm'
```

Comment:
You've clearly worked hard on this, but I think maybe it would be best if we just set the text size based on whether it's a top of the page or mid page hero - it means it looks correct out of the box, and we don't have to worry about people accidentally setting the wrong font size on some pages and making them inconsistent.

Reply:
Thanks, that makes sense. I've updated it to keep 'm' as the default and set it to 'l' only when the position is 'top'.